### PR TITLE
Update links for JavaFX

### PR DESCRIPTION
**Pull request by svenreimers**
- add title for your tube linke
- update JavaFX tutorial links to point to latest Oracle stuff
- added link to openjfx and NetBeans doc

### CODE REVIEW

File: netbeans.apache.org/src/content/kb/docs/java/index.asciidoc [hunk 1]
```diff
@@ -66,10 +66,11 @@ In addition to being a platform for multi-platform desktop applications, the Jav
 
 == JavaFX
 
-- link:https://m.youtube.com/watch?v=8xaRwqcKPSI
-- link:https://docs.oracle.com/javafx/2/get_started/jfxpub-get_started.htm[Getting Started with JavaFX]
+- link:https://m.youtube.com/watch?v=8xaRwqcKPSI[JavaFX by Gluon in Apache NetBeans 11.2]
```
Comment (vieiro) [inline netbeans.apache.org/src/content/kb/docs/java/index.asciidoc]: Hi all,

What about 

`video::8xaRwqcKPSI[youtube, title="JavaFX by Gluon in Apache NetBeans 11.2"]`

That would embed the video directly on the page (don't know if this is a good idea, though)

Comment (svenreimers) [reply netbeans.apache.org/src/content/kb/docs/java/index.asciidoc]: If we agree on directly embedding stuff, this is ok for me - anybody any ideas if there are ASF policies for such things?

Comment (vieiro) [reply netbeans.apache.org/src/content/kb/docs/java/index.asciidoc]: No idea. I'd proceed with the merge as is and then find out, we can always embed it afterwards :-)

Comment (vieiro) [reply netbeans.apache.org/src/content/kb/docs/java/index.asciidoc]: No idea. I'd proceed with the merge as is and then find out, we can always embed it afterwards :-)

**Comment by geertjanw** [comment]
Great, merging.

**Pull request merged**